.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,35 @@
-### What does this PR do?
+### What does this PR do?:
+_Summarize the changes_
 
+### Which issue(s) this PR fixes:
+_Link to github issue(s)_
```

Comment:
Consider putting these as comments (e.g. `<!--Link to github issue(s)-->`) so that they don't show up in PR descriptions directly

Reply:
I'll update these so we're consistent with our other templates

Reply:
done